ts/sdk/src/accounts.rs

```diff
@@ -0,0 +1,52 @@
+use inf1_std::inf1_ctl_core::accounts::{
+    lst_state_list::LstStatePackedList, pool_state::PoolStatePacked,
+};
+use wasm_bindgen::prelude::*;
+
+use crate::interface::{LstState, LstStateList, PoolState, B58PK};
+
+/// @throws
+#[wasm_bindgen(js_name = deserPoolState)]
+pub fn deser_pool_state(bytes: &[u8]) -> Result<PoolState, JsError> {
+    let pool_packed =
+        PoolStatePacked::of_acc_data(bytes).ok_or(JsError::new("Invalid PoolState data"))?;
+    let pool = pool_packed.into_pool_state();
```

Comment:
nit: I like to destructure instead of doing `pool.whatever` to make sure that all fields have been used

```suggestion
    let inf1_std::PoolState { total_sol_value, ... } = pool_packed.into_pool_state();
```